cypress/integration/chi-vue/search-input.spec.js

```diff
@@ -0,0 +1,207 @@
+const SEARCH_INPUTS_DATA_CY = {
+  BASE: '[data-cy="base-search-input-test"]',
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  BUTTON: 'button',
+  DIV: 'div',
+  ICON: 'i',
+  INPUT: 'input'
+};
+const COMPONENT_CLASSES = {
+  CLOSE: '-close',
+  PORTAL: '-icon--primary',
+  SEARCH: '-flat'
+};
+const EVENT_EXAMPLES = {
+  BLUR: 'eventExampleChiBlur',
+  CHANGE: 'eventExampleChiChange',
+  CLEAN: 'eventExampleChiClean',
+  FOCUS: 'eventExampleChiFocus',
+  INPUT: 'eventExampleChiInput',
+  SEARCH: 'eventExampleChiSearch',
+};
+const EVENTS = {
+  BLUR: 'chiBlur',
+  CHANGE: 'chiChange',
+  CLEAN: 'chiClean',
+  FOCUS: 'chiFocus',
+  INPUT: 'chiInput',
+  SEARCH: 'chiSearch',
+};
+
+const checkSizeClass = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.class', value);
+};
+
+const checkAttr = (el, sel, attr, value) => {
+  cy.get(el).find(sel).invoke('attr', attr).should('eq', value);
+};
+
+const checkBooleanAttr = (el, attr, value) => {
+  cy.get(el).invoke('attr', attr).should('eq', value);
+};
+
+const checkPortalAttr = (el, value) => {
+  cy.get(el)
+    .find(`${ELEMENTS.BUTTON}`)
+    .find(`${ELEMENTS.DIV}`)
+    .find(`${ELEMENTS.ICON}`)
+    .should('have.class', value);
+};
+
+const checkValue = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.value', value);
+};
+
+describe('Search Input', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/search-input.html');
+  });
+
+  describe('Base', () => {
+    it('Should exist base search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.BASE}`)
+    });
+  });
+
+  describe('Disabled', () => {
+    it('Should disable search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DISABLED}`)
+        .find(`${ELEMENTS.INPUT}`)
+        .should('have.attr', 'disabled');
+    });
+  });
+
+  describe('Sizes', () => {
+    const sizes = ['sm', 'md', 'lg', 'xl'];
+
+    sizes.forEach((size) => {
+      it(`Should have the class -${size} size`, () => {
+        checkSizeClass(`[data-cy='${size}-search-input-test']`, `-${size}`);
+      });
+    });
+  });
+
+  describe('Attrs', () => {
+    const attrs = ['name', 'placeholder'];
+    const booleanAttrs = ['portal', 'prevent-value-mutation']
```

Comment:
```suggestion
    const booleanAttrs = ['portal']
```

Prevent-value-mutation is not a Chi Vue Search Input prop and should be removed.

Reply:
Solved.